src/modules/identity/controllers/user.controller.ts

```diff
@@ -217,6 +225,14 @@ export class UserController {
     @Res() res: FastifyReply,
     @Body() updatePasswordPayload: UpdatePasswordPayload,
   ) {
+    const expireTime = this.configService.get(
+      "app.emailValidationCodeExpirationTime",
+    );
+    await this.userService.verifyVerificationCode(
+      updatePasswordPayload.email,
+      updatePasswordPayload.verificationCode,
+      expireTime * 2,
```

Comment:
Why are we doing *2

Reply:
to increase the verification code expire time because change password requires lot more time.

Reply:
Then we should take it from a constant.

Reply:
Ok